src/test/java/org/sagebionetworks/bridge/services/Exporter3ServiceTest.java

```diff
@@ -1550,6 +1569,101 @@ public void completeUpload_NoAccount() throws Exception {
+
+        //Mock FileHandle
+        CloudProviderFileHandleInterface markdown = new S3FileHandle();
+        markdown.setStorageLocationId(STORAGE_LOCATION_ID);
+        markdown.setId("exportedTimelinefor" + TestConstants.TEST_STUDY_ID);
+        // How to test file is created correctly? Should be file, not any()
+        when(mockSynapseClient.multipartUpload(any(), eq(STORAGE_LOCATION_ID), eq(false), eq(false)))
+                .thenReturn(markdown);
+        // Mock Wiki
+        V2WikiPage wiki = new V2WikiPage();
+        wiki.setId("wikiFor"  + TestConstants.TEST_STUDY_ID);
+        wiki.setTitle("Exported Timeline for " + TestConstants.TEST_STUDY_ID);
+        wiki.setMarkdownFileHandleId(markdown.getId());
+
+        // Execute and verify output.
+        Exporter3Configuration ex3Config = exporter3Service.exportTimelineForStudy(TEST_APP_ID,
+                TestConstants.TEST_STUDY_ID);
+        assertEquals(ex3Config.getWikiPageId(), "wikiFor" +  TestConstants.TEST_STUDY_ID);
+        assertEquals(ex3Config, study.getExporter3Configuration());
+
+        verifyEx3ConfigAndSynapse(ex3Config, TestConstants.TEST_APP_ID + '/' +
+                TestConstants.TEST_STUDY_ID);
+
+        // Verify call to schedule2Service.
+        verify(mockSchedule2Service).getTimelineForSchedule(TEST_APP_ID, TestConstants.SCHEDULE_GUID);
+        // Verify call to SynapseHelper.
+        verify(mockSynapseHelper).checkSynapseWritableOrThrow();
+        // Verify call to SynapseClient.
+        ArgumentCaptor<V2WikiPage> wikiToCreateCaptor = ArgumentCaptor.forClass(V2WikiPage.class);
+        verify(mockSynapseClient).createV2WikiPage(eq(study.getExporter3Configuration().getProjectId()), eq(ObjectType.ENTITY), wikiToCreateCaptor.capture());
+        System.out.println("********* Project Id: "+ study.getExporter3Configuration().getProjectId());
+    }
+
+    @Test
+    public void exportTimelineForStudy_2ndTime() throws Exception {
+        // Mock Timeline
+        Timeline.Builder builder = new Timeline.Builder();
+        Timeline timeline = builder.withSchedule(mockSchedule2Service.getSchedule(TEST_APP_ID, TestConstants.SCHEDULE_GUID)).build();
+        when(mockSchedule2Service.getTimelineForSchedule(eq(TEST_APP_ID), eq(TestConstants.SCHEDULE_GUID)))
+                .thenReturn(timeline);
+        // Mock File
+        //Mock S3FileHandle
+        CloudProviderFileHandleInterface markdown = new S3FileHandle();
+        markdown.setStorageLocationId(STORAGE_LOCATION_ID);
+        when(mockSynapseClient.multipartUpload(any(), eq(STORAGE_LOCATION_ID), eq(false), eq(false))).thenReturn(markdown);
+        // Mock Wiki
+        V2WikiPage wiki = new V2WikiPage();
+        wiki.setId("wikiFor" + TestConstants.TEST_STUDY_ID);
+        wiki.setTitle("Exported Timeline for " + TestConstants.TEST_STUDY_ID);
+        wiki.setMarkdownFileHandleId(markdown.getId());
+
+        // Execute 1st time export for timeline.
+        exportTimelineForStudy_1stTime();
```

Comment:
Instead of running the first test, we should just set up an Exporter3Config with the attributes that we need, such as wikiPageId.